docs/documenter_helpers.jl

```diff
@@ -23,3 +23,11 @@ else
     end
   end
 end
+
+# We monkey-patch Base.walkdir to use true as default value for follow_symlinks
+# (normally false is the default), in order to "trick" the Documenter code into
+# following those symlinks.
+# See also:
+# https://github.com/JuliaDocs/Documenter.jl/pull/552
+# https://github.com/JuliaLang/julia/blob/master/doc/make.jl#L19
+Base.walkdir(str::String) = Base.walkdir(str; follow_symlinks=true)
```

Comment:
Hmm, we only need this to make Documenter deal with symlinks *to directories*, right ? I think symlinks to *files* work, don't they?

That would suggest an alternative solution: recursively "closing" directories in the source by creating equally named dirs in the destination (if they are not there already) and then adding symlinks for all the files (and deleting stale symlinks). Quite a bit more work (though not really hard).

The main concern I have with the above is that `Oscar.build_doc` loads this code and then `Base.walkdir` is "tainted" in the whole session.

Of course this is existing code, so none of what I say here is relevant for merging this PR!

Anyway, I'll try and see if perhaps Documenter.jl is willing to just use `follow_symlinks=true` in their code, see https://github.com/JuliaDocs/Documenter.jl/pull/2573

Reply:
To be clear this is not meant to block this PR